chroma_core/models/stratagem.py

```diff
@@ -529,6 +531,8 @@ def run(self, args):
             if duration is not None
         ]
 
+        action_list = filter(lambda (label, (mnt_pt, mailbox_file)): path.exists("/var/spool/iml/mailbox/{}".format(mailbox_file)), action_list)
```

Comment:
Utilize `MAILBOX_PATH` in settings. It has this path hardcoded.

Reply:
ah, thanks